src/redux/BookSlice.ts

```diff
@@ -0,0 +1,50 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit'
+import { GoogleBook } from '../api/GoogleBook'
+import { Items, State } from '../types/GoogleBooks'
+import { RootState } from './Store'
+const initialState: State = {
+  searchInput: '',
+  loading: false,
+  error: false,
+  fetchedBooks: [],
+}
+const VOLUMES_SEAECH_PATH = '/volumes?q='
+export const GoogleBooks = createAsyncThunk(
```

Comment:
not a great name for an action creator

It should be a `verySomething`, like `searchBooks`